keel-sql/src/main/resources/db/changelog/20201008-rename-delivery-artifact.yml

```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: rename-delivery-artifact
+      author: lpollo
+      changes:
+        - renameTable:
+            oldTableName: delivery_artifact
+            newTableName: artifact_spec
```

Comment:
What about just `artifact`? that matches the other tables (`resource`, `environment`, etc)

Reply:
I think we should consider renaming the `artifact_uid` columns on `artifact_last_checked`, `delivery_config_artifact`, `environment_artifact_pin`, `environment_artifact_versions`, and `environment_artifact_veto` to `artifact_spec_uid` to avoid confusion

Reply:
+1 to renaming to `artifact`

Reply:
@emjburns @gal-yardeni But it's really _not_ an "artifact". That was the whole point... Another name I discussed with @robfletcher before was `ArtifactDescriptor`, but I think it's uglier than `ArtifactSpec`.

Reply:
Can't you make the same argument about the resource table? It's a resource specification, not the actual resource. I'm just talking about the table name, not the actual class name. For the class I think `ArtifactSpec` makes sense